conda.recipe/post-link.bat

```diff
@@ -1 +1 @@
-"%PREFIX%\bin\python" -m nb_conda.setup install --enable --prefix="%PREFIX%"
+"%PREFIX%\python" -m nb_conda.setup install --enable --prefix="%PREFIX%"
```

Comment:
That's what i changed it from, as it couldn't be found...

On 18:54, Fri, Jan 29, 2016 Damián Avila <notifications@github.com> wrote:

> In conda.recipe/post-link.bat
> <https://github.com/Anaconda-Server/nb_conda/pull/9#discussion_r51331846>:
>
> > @@ -1 +1 @@
> > -"%PREFIX%\bin\python" -m nb_conda.setup install --enable --prefix="%PREFIX%"
> > +"%PREFIX%\python" -m nb_conda.setup install --enable --prefix="%PREFIX%"
>
> we should use %PYTHON% here...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Anaconda-Server/nb_conda/pull/9/files#r51331846>.
>